Error: Can't resolve 'input.scss'

when I insert angular2-emoji into `app.module.ts` in my angular 2.3.1 project, it throws following error:

```
ERROR in ./~/angular2-emoji/input/input.ts
Module not found: Error: Can't resolve 'input.scss' in '<my-app>/node_modules/angular2-emoji/input'
 @ ./~/angular2-emoji/input/input.ts 80:21-42
 @ ./~/angular2-emoji/module.ts
 @ ./~/angular2-emoji/index.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi main
```

but the file `<my-app>/node_modules/angular2-emoji/input/input.scss` actually exists.

--- a/src/lib/input/input.ts
+++ b/src/lib/input/input.ts
@@ -4,8 +4,8 @@ import { EmojiUtil } from '../util/util';
 
 @Component({
   selector: 'emoji-input',
-  templateUrl: 'input.html',
-  styleUrls: ['input.scss']
+  templateUrl: './input.html',
+  styleUrls: ['./input.scss']
 })
 export class EmojiInputComponent implements OnInit, OnChanges {
 

